File: python/svix/models/integration_out.py
# this file is @generated
import typing as t
from datetime import datetime

from .common import BaseModel


class IntegrationOut(BaseModel):
    name: str

    id: str
    """The Integration's ID."""

    created_at: datetime

    updated_at: datetime

    feature_flags: t.Optional[t.List[str]] = None
    """The set of feature flags the integration has access to."""

File: kotlin/lib/src/main/kotlin/models/IntegrationOut.kt
// This file is @generated
package com.svix.kotlin.models

import kotlin.time.Instant
import kotlinx.serialization.Serializable

@Serializable
data class IntegrationOut(
    val name: String,
    /** The Integration's ID. */
    val id: String,
    val createdAt: Instant,
    val updatedAt: Instant,
    /** The set of feature flags the integration has access to. */
    val featureFlags: Set<String>? = null,
)
